source/types.ts

```diff
@@ -114,7 +114,7 @@ export interface KnownTarget {
 
 export interface Target {
   tabId: number;
-  frameId?: number;
+  frameId?: number | "allFrames";
```

Comment:
- `tabs.sendMessage` treats a missing `frameId` as a request to message all frames. 
- `webext-messenger` treats it as a request to message the top frame. I feel this was a good default.

Instead of changing this behavior or adding `allFrames: true`, I just added a new value to `frameId`. This avoids unnecessary type refactoring to ensure the user never passes both `frameId` and `allFrame` properties. Altering `Target` brought a cascade of other issues.